### DISCUSSION TITLE
Google Search Console Integration Step 4 Domain Verification

### DISCUSSION THREAD

**Discussion by CreateSean** [plausible/analytics#3773] [Self-Hosted Support]
I've got my self hosted analytics set up and everything is working. However I am unable to finish integrating the search console with these [instructions](https://plausible.io/docs/self-hosting-configuration#google-search-integration).

`Finally, return to APIs & Services, and select "Domain verification". If you can't see the "Domain verification" menu item, open the navigation ("hamburger") menu in the top left corner and select "APIs & Services". "**Domain verification**" is a menu item below that.`

There is **no** domain verification option. Please advise

![image](https://github.com/plausible/analytics/assets/6899490/273527a1-f151-4ca3-9385-5cbd62732c82)

**Comment by ruslandoga** [comment]
👋 @CreateSean 

Domain verification has been moved to [Google Search Console pages.](http://search.google.com/u/1/search-console/welcome)

Once there, either ensure that you've already verified your domain by checking the properties in the Select property dropdown on the left or pick one of the two ways to verify it.

<img src="https://github.com/ruslandoga/plausible-self-hosted/raw/master/images/4-search-console-new.png" />

**Comment by CreateSean** [comment]
So it appears that it's good

![image](https://github.com/plausible/analytics/assets/6899490/85403688-25e7-428c-90b9-051fb26c9cf0)


However in plausible I still get prompted to complete the extra step.

![image](https://github.com/plausible/analytics/assets/6899490/f3ee8a78-0747-4ffe-836e-1fb60cd85229)


I checked the docs here: https://plausible.io/docs/google-search-console-integration#allow-plausible-analytics-to-access-your-search-console

and it's telling me to go to the integrations page of the site settings in plausible. But self-hosted plausible doesn't have integrations option.

![image](https://github.com/plausible/analytics/assets/6899490/35fcf6e1-be99-46bd-8732-574b73cad3c5)

**Reply by ruslandoga** [reply]
> But self-hosted plausible doesn't have integrations option.

It's the "Search console" in v2.0.0.

> However in plausible I still get prompted to complete the extra step.

Maybe this guide would be helpful: https://github.com/ruslandoga/plausible-self-hosted?tab=readme-ov-file#integrate-with-google-search

Note that you need to enable the "Google Search Console API"
